lib/Page.js

```diff
@@ -60,6 +60,7 @@ function Page(pageConfig) {
   this.frontMatter = {};
   this.headings = {};
   this.includedFiles = {};
+  this.searchable = pageConfig.searchable;
```

Comment:
Let's move this line below line 46, as the variables that are using `pageConfig` are all initialized there.